### ISSUE TITLE
Segmentation violation on testing suite when in parallel mode

### ISSUE THREAD

**Issue by sts-ryan-holton** [laravel/framework#49445]
### Laravel Version

10.38.0

### PHP Version

8.2.13

### Database Driver & Version

MySQL 8.0

### Description

I've just updated my Laravel project. I'm using a bunch of feature tests in my project, I'm also using `brianium/paratest` ^7.3.  When I run `php artisan test --parallel --stop-on-failure` the tests instantly fail with:

> The test "PARATEST='1' TEST_TOKEN='10' UNIQUE_TEST_TOKEN='10_6582df3044194' tests/Feature/Helpers/CsvExportSetupTest.php"  
   failed.                                                                                                                   
                                                                                                                             
  

### Steps To Reproduce

This didn't happen prior to this version. Also, just running `php artisan test` works normally.

**Comment by crynobone** [comment]
Hey there, thanks for reporting this issue.

We'll need more info and/or code to debug this further. Can you please create a repository with the command below, commit the code that reproduces the issue *as one separate commit* on the main/master branch and share the repository here?

Please make sure that you have the [latest version of the Laravel installer](https://github.com/laravel/installer) in order to run this command. Please also make sure you have both Git & [the GitHub CLI tool](https://cli.github.com/) properly set up.

    laravel new bug-report --github="--public"

Do not amend and create a separate commit with your custom changes. After you've posted the repository, we'll try to reproduce the issue.

Thanks!

**Comment by crynobone** [comment]
Hey there,

We're closing this issue because it's inactive, already solved, old, or not relevant anymore. Feel free to open up a new issue if you're still experiencing this problem.

**Comment by chrispmcgee** [comment]
Getting the exact same problem with Laravel 10.45.1 and PHP 8.2.15. I'm running within WSL on Windows 11. Normal `sail artisan test` works flawlessly.
